frontend/src/components/Uploader/ProvidersUploader.tsx

```diff
@@ -49,13 +52,13 @@ function onClearInputs(store: Store) {
  * Aliases are compared to column names as lowerCase()
  */
 const COLUMNS = [
-  { aliases: ['Address'], required: true },
-  { aliases: ['City'], required: true },
-  { aliases: ['State'], required: true },
-  { aliases: ['Postal Code', 'Zip', 'ZipCode', 'Zip Code'], required: true },
+  { aliases: ['Address'], required: (ENV !== 'PRD') },
+  { aliases: ['City'] },
+  { aliases: ['State'] },
+  { aliases: ['Postal Code', 'Zip', 'ZipCode', 'Zip Code'] },
   { aliases: ['Health Center Type'] },
-  { aliases: ['Latitude'] },
-  { aliases: ['Longitude'] },
+  { aliases: ['Latitude', 'lat'], required: (ENV === 'PRD') },
```

Comment:
how about 'latitude' and 'longitude' too

Reply:
They are already accepted, these are case insensitive